### PR TITLE
add security context to plugin resources template

### PR DESCRIPTION
**Pull request by jgbernalp**

### CODE REVIEW

File: logging-view-plugin-resources.yml [hunk 1]
```diff
@@ -78,10 +78,19 @@ spec:
...
+              drop:
+                - ALL
           ports:
             - containerPort: 9443
               protocol: TCP
           imagePullPolicy: Always
+          resources:
+            requests:
+              cpu: 10m
+              memory: 50Mi
```
Comment (periklis) [inline logging-view-plugin-resources.yml:93]: Really that low? What's the consideration? Because we serve static files from nginx only?

**Review by periklis — COMMENTED** [review]
Overall looks good to me

File: logging-view-plugin-resources.yml (hunk 1 above)
Comment (jgbernalp) [reply logging-view-plugin-resources.yml:93]: yes, we are not doing any other backend work now (might change in the future)

Comment (jgbernalp) [reply logging-view-plugin-resources.yml]: 👍🏽

**Review by periklis — APPROVED** [review]

**Pull request merged**